maleka_brown/nba_challenge/nba_challenge.py

```diff
@@ -0,0 +1,57 @@
+print(f'Three point shots attempted by Jamal {Jamal_Murray_attempted}.' )
+print(f'Three point shots attempted by Fred {Fred_VanVleet_attempted}.')
+print(f'Three point shots attemoted by James{James_Harden_attempted}.\n')
+
+
+#three point shots made and three point shots attempted
+
+Jamal_shots_percentage = int(Jamal_Murray/Jamal_Murray_attempted*100)
+Fred_shots_percentage = int(Fred_VanVleet/Fred_VanVleet_attempted*100)
+James_shots_percentage = int(James_Harden/James_Harden_attempted*100)
+
+print(f'Jamal three point shots percentage {Jamal_shots_percentage}%.')
+print(f'Fred three point shots percentage {Fred_shots_percentage}%.')
+print(f'James three point shots percentage {James_shots_percentage}%.\n')
+
+# About the Lakers
+
+print('The Lakers went all in this offseason and swung a deal for former Pelicans forward Anthony Davis. \
+\nThey sent a package of Brandon Ingram, Lonzo Ball, Josh Hart, and 3 first-round picks to New Orleans to land Davis. \
+\nThose three have made good developments with the Pelicans, especially Brandon Ingram. \
+\nBut, the deal is still a huge win for the Lakers as Lebron, Davis, and company have put together an incredible season. \
+\nLos Angeles has ridden James and Davis, along with a supporting cast built around them, to the second-best record in the NBA. \
+\nThe Lakers ended the season atop the Western Conference with a record of 49-14. \
+\nThey were narrowly behind the Bucks for the best record in the league. \
+\nDavis proved to the final piece necessary for the Lakers to rebound from missing the playoffís last year. \
+\nLos Angeles was a dominant club on both sides of the ball and are in a position to have another successful year next season.')
+
+about_lakers = 'The Lakers went all in this offseason and swung a deal for former Pelicans forward Anthony Davis. \
+\nThey sent a package of Brandon Ingram, Lonzo Ball, Josh Hart, and 3 first-round picks to New Orleans to land Davis. \
+\nThose three have made good developments with the Pelicans, especially Brandon Ingram. \
+\nBut, the deal is still a huge win for the Lakers as Lebron, Davis, and company have put together an incredible season. \
+\nLos Angeles has ridden James and Davis, along with a supporting cast built around them, to the second-best record in the NBA. \
+\nThe Lakers ended the season atop the Western Conference with a record of 49-14. \
+\nThey were narrowly behind the Bucks for the best record in the league. \
+\nDavis proved to the final piece necessary for the Lakers to rebound from missing the playoffís last year. \
+\nLos Angeles was a dominant club on both sides of the ball and are in a position to have another successful year next season.'
+print(about_lakers.upper())
+
+lakers_are_the_best = True
+print(f'Lakers are the best team in the NBA is {lakers_are_the_best}.')
+lakers_are_the_best = int(lakers_are_the_best)
+print(lakers_are_the_best)
+lakers_are_the_best = float( lakers_are_the_best)
+print(lakers_are_the_best)
+
```

Comment:
The last part of the challenge was to compare `str(Jamal_shots_percentage)` and `int(Jamal_shots_percentage)`. When you run both of these functions the `str` function will return the actual figure i.e. `.463432` whereas `int` will return `0`